ui/components/NpmUi.jsx

```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react';
-import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import React, {Component} from 'react';
+import {Tab, Tabs, TabList, TabPanel} from 'react-tabs';
```

Comment:
only thing I doubt about is bracketSpacing, can we have it on? No strong feelings though

Reply:
Oh sure, I only turned it off because there were a few imports like this.